Viper Practice/src/controllers/fruites detail/FruitDetailViewController.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  FruitDetailViewController.swift
+//  Viper Practice
+//
+//  Created by Huzaifa-Umer on 22/04/2024.
+//
+
+import UIKit
+
+class FruitDetailViewController: 
+    UIViewController,
+    UITableViewDataSource,
+    FruitDetailViewProtocol {
+    
+    
+
+    @IBOutlet weak var tableView: UITableView!
+    var presenter : FruitDetailPresenterProtocol?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        title = Strings.Title.fruiteDetail
+        tableView.dataSource = self
+        presenter?.loadData()
+    }
+    
+    // MARK: - FruitDetailViewProtocol
+    
+    func displayFruitDetail() {
+        tableView.reloadData()
+    }
+    
+    // MARK: - UITableViewDataSource
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 2
+    }
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return section == 0 ? 4 : 5
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        switch indexPath.section {
+        case 0:
+            let cell = tableView.register(FruitDetailRowTableViewCell.self, indexPath: indexPath)
```

Comment:
create a configure method inside cell and pass the fruit object, all these logics should be there